src/plugins/azgenerator/TemplateAzureCliParams.ts

```diff
@@ -171,8 +171,38 @@ function getCommandBody(model: CodeModelAz, needUpdate: boolean = false, needGen
 
                         argument = "        c.argument('" + parameterName + "'";
                         argument += ", options_list=['--" + parameterName.substr(0, parameterName.length - 1) + "']";
+                    } else if (parameterName.endsWith('name') && parameterName.replace(/_name|_/g, '') == model.CommandGroup_DefaultName.toLowerCase()) {
```

Comment:
shouldn't be endsWith('_name')?

Reply:
no. this is to fix issue 282. to automatically add options_list in main resource name

Reply:
I use the comamnd group name and parameter name to match whether this parameter is main resource name or not.

Reply:
ok